LiskHQ/lisk
# Update system headers and revert node network height changes - Closes #1778

### What was the problem?
NetworkHeight shows always return zero as the system height was not updated when the block was broadcasted.
### How did I fix it?
Change the system `update` implementation to pass the block to use the height to update system `height` and reverted the node `getStatus` network height logic to 0 if the network height is null or undefined.
### How to test it?
Run two nodes with one forging and one non-forging and make API request to non-forging block `http://localhost:4001/api/node/status` you should see the network height.
### Review checklist

* The PR solves #1778 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
